lib/Resizable.js

```diff
@@ -182,6 +186,27 @@ export default class Resizable extends React.Component<Props, State> {
       const canDragX = (this.props.axis === 'both' || this.props.axis === 'x') && ['n', 's'].indexOf(axis) === -1;
       const canDragY = (this.props.axis === 'both' || this.props.axis === 'y') && ['e', 'w'].indexOf(axis) === -1;
 
+      // Check if handle's position has moved since last resizeHandler call - adjust deltas accordingly
+      if (this.draggingNode == null && e.target instanceof HTMLElement) {
+        this.draggingNode = e.target;
+      }
+      if (this.draggingNode instanceof HTMLElement) {
+        const handleRect = this.draggingNode.getBoundingClientRect();
+        if (this.lastTop != null && this.lastLeft != null) {
+          const deltaLeftSinceLast = handleRect.left - this.lastLeft;
+          const deltaTopSinceLast = handleRect.top - this.lastTop;
+          
+          if (canDragX && axis.indexOf('w') > -1) {
```

Comment:
The allowable values for axis are:
```js
['s', 'w', 'e', 'n', 'sw', 'nw', 'se', 'ne']
```
So this could be more clear as `axis[axis.length - 1] === 'w'`, similar to https://github.com/STRML/react-resizable/blob/a1fff416339df13b15b23044b026ebbe376a2b9d/lib/Resizable.js#L97.

This of course would be a great use case for `String#endsWith()` but we are trying to avoid prototype methods that [aren't supported in IE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith).

Reply:
Good point, I've made this change now